descriptors/ModuleDescriptor-template.json

```
@@ -38,6 +38,11 @@
           "methods": ["POST"],
           "pathPattern": "/loan-storage/loans/anonymize/{userId}",
           "permissionsRequired": ["circulation-storage.loans.collection.anonymize.user.post"]
+        },
+        {
+          "methods": ["POST"],
+          "pathPattern": "/loan-storage/anonymize-loans",
+          "permissionsRequired": ["circulation-storage.anonymizeloans.post"]
```

Personally, I'd name this as similar to the endpoint as possible, using hyphens between words.

For example,
```suggestion
          "permissionsRequired": ["circulation-storage.anonymize-loans.post"]
```

---

This has been reworked.